src/magicbox/SuggestionsManager.ts

```diff
@@ -65,16 +103,15 @@ export class SuggestionsManager {
     });
 
     this.suggestionsListbox = this.buildSuggestionsContainer();
-    this.suggestionsPreviewContainer = this.initPreviewForSuggestions(this.suggestionsListbox);
-    $$(this.element).append(this.suggestionsPreviewContainer.el);
+    $$(this.element).append(this.suggestionsListbox.el);
```

Comment:
This method was responsible for wrapping the suggestions container in a new top-level container along with the search result previews container. However, doing this in the constructor involved checking if previews existed in the page, causing issues when lazy loading. The suggestions container is now added as-is and only wrapped once search result previews are received for the first time.